terasoluna-gfw-functionaltest-web/pom.xml

```diff
@@ -93,10 +93,9 @@
       <groupId>org.postgresql</groupId>
       <artifactId>postgresql</artifactId>
     </dependency>
-    <dependency>
-      <groupId>com.oracle</groupId>
-      <artifactId>ojdbc7</artifactId>
-    </dependency>
+    <!-- Once you have installed ojdbc7.jar into local repository, please remove this comment out. -->

```

Comment:
This fix is only 1.0.x.
`Begin Database` is deleted after 5.0.x by issue #196 .
OK.